analyzers/src/SonarAnalyzer.Common/SymbolicExecution/Roslyn/OperationProcessors/Invocation.cs

```diff
@@ -19,17 +19,18 @@
  */
 
 using Microsoft.CodeAnalysis;
+using SonarAnalyzer.Helpers;
 using SonarAnalyzer.SymbolicExecution.Constraints;
 using StyleCop.Analyzers.Lightup;
 
 namespace SonarAnalyzer.SymbolicExecution.Roslyn.OperationProcessors;
 
-internal class Invocation : SimpleProcessor<IInvocationOperationWrapper>
+internal class Invocation : MultiProcessor<IInvocationOperationWrapper>
```

Comment:
All these classes should be sealed (Maybe a follow up PR).

Reply:
Will do here, I need to rebase anyway